packages/engine/src/FeatureFlagsState.tsx

```diff
@@ -37,7 +38,7 @@ export const FeatureFlagsState = defineState({
   },
   useEnabled(flagName: string) {
     const state = useHookstate(getMutableState(FeatureFlagsState)[flagName]).value
-    return typeof state === 'boolean' ? state : true
+    return typeof state === 'boolean' ? state : !isUndefined(state)
```

Comment:
Instead of this you can simply make it `false`.

Though @HexaField with this change features will be `off` by default in contrast to how they currently are `on` by default.

Reply:
we need to automatically seed all flags if we are going to turn the off by default. local dev should have all flags on by default, otherwise it'll be a massive hassle any time someone reinits.

Reply:
@mikeplascdev the point of concern in the issue was, maybe hide the feature as data regarding feature flag is being fetched from backend. And then if an item is not found in feature flags data then it means its enabled unless its explicitly turned off.

Reply:
Oh I see. In that case yes we should go ahead and assume they are disabled by default, (else condition false) and add seeding for local dev

Reply:
We also do want to replace FeatureFlagState with in-place useFind with the flagname in the query, since feature flags will paginate and we are getting close to having 10 settings, so we should do that soon.

Reply:
@mikeplascdev marked this PR as draft as above needs to be addressed